pkg/ibmvpc/vpc.go

```diff
@@ -115,12 +113,16 @@ func (v *VPC) subnets() []*Subnet {
 	return v.subnetsList
 }
 
+// Subnet implements vpcmodel.Subnet interface
 type Subnet struct {
 	vpcmodel.VPCResource
-	nodes             []vpcmodel.Node
-	connectivityRules *vpcmodel.ConnectivityResult // allowed connectivity between elements within the subnet
```

Comment:
Same question - why was this removed? what replaces it?

Reply:
same here